node-fs-readfile/read-any.js

```diff
@@ -0,0 +1,22 @@
+const filename = process.argv[2];
+
+const fs = require('fs');
+
+if (filename === 'cunningham.txt') {
+  fs.readFile(`../rc0122-code-solutions/node-fs-readfile/${filename}`, 'utf8', callBack);
+} else if (filename === 'dijkstra.txt') {
+  fs.readFile(`../rc0122-code-solutions/node-fs-readfile/${filename}`, 'utf8', callBack);
+} else if (filename === 'hipster-ipsum.txt') {
+  fs.readFile(`../rc0122-code-solutions/node-fs-readfile/${filename}`, 'utf8', callBack);
+} else if (filename === 'hopper.txt') {
+  fs.readFile(`../rc0122-code-solutions/node-fs-readfile/${filename}`, 'utf8', callBack);
+} else {
+  console.log('INVALID');
+}
```

Comment:
Instead of doing all of this it would be better to just use the variable as the filename:
```JAVASCRIPT
fs.readFile(filename, 'utf8', callBack);
```